sources/packages/web/src/components/aest/StudentApplicationOfferingChangeDetails.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-card>
```

Comment:
we had a component called `body-header-container` can't we use it here?

Reply:
<img width="1919" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/172b6aa3-bfd1-4006-b6f0-6e7e6e2222ad">

I have updated it to use `body-header-container` but the padding as shown above cannot be adjusted unless I make the padding adjustment in the `body-header-container`